src/eclipse/eclipse.importorder

```diff
@@ -0,0 +1,7 @@
+#Organize Import Order
+#Wed Apr 26 12:53:22 EDT 2017
+4=\#
+3=org.springframework
+2=
+1=javax
+0=java
```

Comment:
OK. When I switched off:

![image](https://user-images.githubusercontent.com/1071590/26982696-30760350-4d07-11e7-9033-e66b75d1fbf1.png)

Everything has started to work properly.
Would be great to mention that in the `CONTRIBUTING.adoc`